src/components/CSVReference.vue

```diff
@@ -35,20 +35,6 @@
             </p>
           </div>
 
-          <br />
-          <v-btn
-            class="flex ma-2"
-            target="_blank"
-            append-icon="mdi-open-in-new"
-            href="https://github.com/internetofwater/geoconnex.us/blob/master/namespaces/iow/demo.csv"
-            >1:1 example</v-btn
-          >
-          <v-btn
-            target="_blank"
-            append-icon="mdi-open-in-new"
-            href="https://github.com/internetofwater/geoconnex.us/blob/master/namespaces/usgs/monitoring-location/monitoring-location.csv"
-            >1:N example</v-btn
```

Comment:
I think we could also keep this still since it is easier to have a direct link there instead of -> docs -> example but either way is probably fine. Just worried that others might not see the example

Maybe I should make the docs have the 1:1 1:N examples in the docs itself instead of linking to Github.

Reply:
Thats a good idea. Mostly was that all the other hyperlinks are <a> elements and these are buttons which made me think to change. I will revert